nilearn/decomposition/tests/test_dict_learning.py

```diff
@@ -155,7 +155,6 @@ def test_masker_attributes_with_fit(canica_data, mask_img):
     dict_learning = DictLearning(n_components=3, mask=mask_img, random_state=0)
     dict_learning.fit(canica_data)
 
-    assert dict_learning.mask_img_ == mask_img
```

Comment:
removing as mask_img_ may now be different from mask_img (it may be a binarized version of it)